src/components/events/Event.js

```diff
@@ -1,7 +1,34 @@
+import Link from "next/link";
 import React from "react";
-
-const Event = () => {
-  return <div>Event</div>;
+import { FiExternalLink } from "react-icons/fi";
+const Event = ({
```

Comment:
pass in these variables in the page.js where you are testing. Not in the main file